src/main/java/mops/controller/dto/QuestionPollLifecycleDto.java

```diff
@@ -0,0 +1,13 @@
+package mops.controller.dto;
+
+import java.util.Date;
+
+import lombok.AllArgsConstructor;
+import lombok.Data;
+
+@Data
+@AllArgsConstructor
+public class QuestionPollLifecycleDto {
+    private Date start;
```

Comment:
Date zu LocalDateTime (kein konstruktor für LocalDateTime Objekte initialisierung ist statisch LocalDateTime.of(year, month, day, hour, min, sec).